ui/components/User.js

```diff
@@ -67,41 +76,28 @@ const User = (props) => {
   };
 
   const handleGetToken = () => {
-    dataFetch(
-      '/api/token',
-      { credentials: 'same-origin' },
-      (data) => {
+    triggerGetToken()
+      .unwrap()
+      .then((data) => {
         exportToJsonFile(data, 'auth.json');
-      },
-      (error) => ({ error }),
-    );
+      });
   };
 
-  useEffect(() => {
-    dataFetch(
-      '/api/user',
-      {
-        credentials: 'same-origin',
-      },
-      (userData) => {
-        setUser(userData);
-        dispatch(updateUser({ user: userData }));
-      },
-      (error) => ({
-        error,
-      }),
-    );
-  }, [dispatch]);
-
-  useEffect(() => {
-    const { capabilitiesRegistry } = props;
-    if (!capabilitiesLoaded && capabilitiesRegistry) {
-      setCapabilitiesLoaded(true); // to prevent re-compute
-      setAccount(
-        ExtensionPointSchemaValidator('account')(capabilitiesRegistry?.extensions?.account),
-      );
-    }
-  }, [capabilitiesRegistry, capabilitiesLoaded]);
+  if (!userLoaded && isGetUserSuccess) {
+    props.updateUser({ user: userData });
+    setUserLoaded(true);
+  } else if (isGetUserError) {
+    console.error('Error fetching user: ', getUserError?.data);
+  }
+
+  if (isTokenError) {
+    console.error('Error fetching token: ', tokenError?.data);
+  }
```

Comment:
Hey @dottharun we can use `notify` hook here so that this issue gets displayed as toaster in UI, this issue needs user attention.

You can find implementation of that in Meshery codebase

Reply:
Maybe in new iteration PR you can add that

Reply:
https://github.com/meshery/meshery/pull/10270#discussion_r1505087614

Reply:
https://github.com/meshery/meshery/pull/10270#discussion_r1505087614

Reply:
@sudhanshutech do we want to use Notification center here?

Reply:
Yes please we want to use Notification center here, it always important to UI state with user, we can ignore those errors which doesn't block or affect UI, otherwise it's always good to show error to user